src/engines/misc/sends/screen.cpp

```diff
@@ -26,304 +30,117 @@ namespace otto::engines::sends {
 
   void Screen::draw(nvg::Canvas &ctx)
   {
-    using namespace core::ui::vg;
-    ctx.font(Fonts::Norm, 35);
-
-    // constexpr float x_pad = 30;
-
-    // constexpr float y_pad = 50;
-    // constexpr float space = (height - 2.f * y_pad) / 3.f;
-
-    // sends/sendsIcon
-
-    // sends/sendsIcon/redLine
-    ctx.save();
-    ctx.beginPath();
-    ctx.moveTo(38.4, 112.2);
-    ctx.lineTo(72.2, 78.4);
-    ctx.lineWidth(6.0);
-    ctx.strokeStyle(Colours::Gray70);
-    ctx.lineCap(LineCap::ROUND);
-    ctx.lineJoin(LineJoin::ROUND);
-    ctx.stroke();
-
-    // sends/sendsIcon/Send1Arrow
-    ctx.beginPath();
-    ctx.moveTo(62.4, 78.5);
-    ctx.lineTo(72.2, 78.4);
-    ctx.lineTo(72.2, 88.3);
-    ctx.strokeStyle(Colours::Blue);
-    ctx.stroke();
-
-    // sends/sendsIcon/yellowLine
-    ctx.beginPath();
-    ctx.moveTo(38.4, 127.7);
-    ctx.lineTo(72.3, 161.6);
-    ctx.strokeStyle(Colours::Gray70);
-    ctx.stroke();
-
-    // sends/sendsIcon/Send2Arrow
-    ctx.beginPath();
-    ctx.moveTo(72.2, 151.7);
-    ctx.lineTo(72.2, 161.6);
-    ctx.lineTo(62.4, 161.5);
-    ctx.strokeStyle(Colours::Green);
+    // circles
```

Comment:
I cant really tell if this includes the background lines, does it?

Reply:
Not yet, but I will add them now